vignettes/sorting_pruning.Rmd

```diff
@@ -121,18 +140,48 @@ pruned4
 
 ## Sorting
 
-Sorting an rtable is done **at a path** and recursively, meaning a sort operation will occur at a particular location within the table, and the subtables (children) will both be reordered themselves and potentially have their own children reordered as well.
+Here, understanding pathing is very important. Indeed, sorting an `rtable` is done **at a path** and recursively, meaning a sort operation will occur at a particular location within the table, and the subtables (children) will both be reordered themselves and potentially have their own children reordered as well. Sorting can be done with `sort_at_path()` function, and it has been designed to need a specific path 
+and scoring function.
+
+A *score function* accepts a subtree or `TableRow` and returns a single numeric value. Within the context currently being sorted, the subtrees are then reordered by the value of the score function. Importantly, "content" (`ContentRow`) and "values" (`DataRow`) need to
+be treated differently in the scoring function as they are retrieved also differently. For example, the internally defined `cont_n_allcols()` will work with calling content value of a specific subtable (retrieved _always_ *one level* under `path`) with the access function `content_table()`. Note that this function would fail if the initial node in the subtable is NOT a content node (`ContentRow`, produced by `summarize_row_groups()`). If we call `cont_n_allcols` without parenthesis we can see its construction and how values are retrieved.
```

Comment:
```suggestion
A *score function* accepts a subtree or `TableRow` and returns a single numeric value. Within the context currently being sorted, the subtrees are then reordered by the value of the score function. Importantly, "content" (`ContentRow`) and "values" (`DataRow`) need to
be treated differently in the scoring function as they are retrieved also differently. For example, the internally defined `cont_n_allcols()` will work with calling the content value of a specific subtable (retrieved _always_ *one level* under `path`) with the access function `content_table()`. Note that this function would fail if the initial node in the subtable is NOT a content node (`ContentRow`, produced by `summarize_row_groups()`). If we call `cont_n_allcols` without parenthesis we can see its construction and how values are retrieved.
```